### PR TITLE
Update to latest db_connection

### PR DESCRIPTION
**Pull request by josevalim**
/cc @fishcakez

### CODE REVIEW

File: lib/mariaex.ex [hunk 1]
```diff
@@ -69,9 +69,11 @@ defmodule Mariaex do
        backoff and to stop (see `:backoff`, default: `:jitter`)
     * `:transactions` - Set to `:strict` to error on unexpected transaction
       state, otherwise set to `naive` (default: `:naive`);
-    * `:pool` - The pool module to use, see `DBConnection`, it must be
-      included with all requests if not the default (default:
-      `DBConnection.Connection`);
+    * `:idle` - Either `:active` to asynchronously detect TCP disconnects when
+      idle or `:passive` not to (default: `false`);
```
Comment (fishcakez) [inline lib/mariaex.ex]: should be default: `false`, I see the same typo is made in postgrex :P

Comment (fishcakez) [inline lib/mariaex.ex]: Er should not be default: `false` but default: `:passive`

File: mix.lock [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  "connection": {:hex, :connection, "1.0.2"},
-  "coverex": {:hex, :coverex, "1.4.3"},
-  "db_connection": {:hex, :db_connection, "0.2.0"},
-  "decimal": {:hex, :decimal, "1.1.0"},
-  "hackney": {:hex, :hackney, "1.3.2"},
-  "httpoison": {:hex, :httpoison, "0.7.2"},
-  "idna": {:hex, :idna, "1.0.2"},
-  "poison": {:hex, :poison, "1.4.0"},
-  "ssl_verify_hostname": {:hex, :ssl_verify_hostname, "1.0.5"}}
+%{"backoff": {:hex, :backoff, "1.1.1", "9c60418f672549304c565ae789282c597c7cc2a1eca1e6125a38bf2dd3bd9fad", [:rebar3], []},
```
Comment (fishcakez) [inline mix.lock]: I think we can unlock this as no longer use it in db_connection - unless used by mariaex

**Comment by josevalim** [comment]
Force pushed.

**Comment by liveforeverx** [comment]
Thank you!

**Pull request merged**